AppInspector.CLI/CLICmdOptions.cs

```diff
@@ -58,14 +54,29 @@ public class AnalyzeCommandOptions : AllCommandOptions
         [Option('k', "file-path-exclusions", Required = false, HelpText = "Exclude source files (none|default: sample,example,test,docs,.vs,.git)", Default = "sample,example,test,docs,.vs,.git")]
         public string FilePathExclusions { get; set; }
 
-        [Option('x', "console-verbosity", Required = false, HelpText = "Console verbosity [high|medium|low|none]", Default = "medium")]
-        public string ConsoleVerbosityLevel { get; set; }
+        [Option('f', "output-file-format", Required = false, HelpText = "Output format [html|json|text]", Default = "html")]
+        public new string OutputFileFormat { get; set; }
+
+        [Option('e', "text-format", Required = false, HelpText = "Match text format specifiers", Default = "Tag:%T,Rule:%N,Ruleid:%R,Confidence:%X,File:%F,Sourcetype:%t,Line:%L,Sample:%m")]
+        public string TextOutputFormat { get; set; }
+
+        [Option('t', "tag-output-only", Required = false, HelpText = "Output only identified tags", Default = false)]
+        public bool SimpleTagsOnly { get; set; }
+
+        public CLIAnalyzeCmdOptions()
+        {
+            OutputFileFormat = "html";
+            MatchDepth = "best";
+            IgnoreDefaultRules = false;
+            SimpleTagsOnly = false;
```

Comment:
Why set this default value twice?  See line 63.

Reply:
Because found that CommandLineParser will not set them if a new override is provided with a different default than the base.  Had to counter by putting in manually.